canvas_modules/harness/test_resources/parameterDefs/oneofselect_paramDef.json

```diff
@@ -395,6 +436,22 @@
           "default": "oneofselect filtered to have values 'red', 'blue', and 'green'"
         }
       },
+      {
```

Comment:
Was this suppose to have `custom_value_allowed`?

Reply:
Yes it is, just fixed this